models/track_analytics.go

```diff
@@ -0,0 +1,9 @@
+package models
+
+// TrackEvent ...
+type TrackEvent struct {
+	ID         string                 `json:"id"`
```

Comment:
I assume this ID is a UUID. I checked the code usages of this field and we are not validating the value of this string, just passing it on to PubSub. I'd rather change this `string` type to some kind of `UUID` type and validate the value in `Tracker.Send()` just in case.

Reply:
Fun fact: golangci-lint doesn't support 1.18 yet: https://github.com/golangci/golangci-lint/issues/2649

Reply:
Added validation